README.md

```diff
@@ -115,14 +127,41 @@ For requesting the actor to the server you have to start the tracker. This way a
 TrackerAsset.Instance.Start();
 ```
 
+### Tracker Login and Start (asynchronous)
+
+To use the tracker asynchronously, you should use the methods LoginAsync and StartAsync.
+
+#### Tracker LoginAsync
+
+Tracker LoginAsync is not really necesary. If you are a developer, and you're logged into the Analytics server as a developer, you're going to receive traces either the player has logged or not. If you're a teacher, and you want to see traces in your activities, you can configure the activity to allow anonymous users.
+
+If you are a teacher and you want to use logged students, you have to add the students to the class and then ask them for credentials into your game and log them into the system using `TrackerAsset.Instance.LoginAsync(String username, String password)` function.
```

Comment:
The LoginAsync method uses an Action<bool> as third parameter that will be used to notify when the login attempt has finished and wether it was a success or not.